lib/scope.js

```diff
@@ -650,9 +650,34 @@ module.exports = {
     }
   },  // Ends scope.setLanguage()
 
-  normalizeTable: async function ( scope, { var_data, from_story_table=true }) {
-    /* Return data derived from cucumber variable-setting data.
-     * Table must have headers. Arrays are not supported. */
+  normalizeTable: async function ( scope, { var_data=null, raw_var_data=null, from_story_table=true }) {
+    /* Return data derived from cucumber variable-setting data, from either an array or object format */
+
+    if (!var_data && ! raw_var_data) {
+      let no_data_msg = `ALKiln Internal error: normalizeTable called without var_data or raw_var_data`;
+      await scope.addToReport( scope, { type: `error`, value: no_data_msg });
+      throw Error(bad_table_msg);
```

Comment:
Maybe there should be a test for this too.

```suggestion
      throw Error(no_data_msg);
```

Reply:
Add to the message that they should explicitly open an issue if they see this